src/main/controllers/features/unincorporated/businessAddressAutoLookupController.ts

```diff
@@ -29,6 +30,7 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 export const post = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as any as Session;
     const acspData : ACSPData = session?.getExtraData(USER_DATA)!;
+    const acspData1 : AcspData = session?.getExtraData(USER_DATA)!;
```

Comment:
is this intentional? using acspData1?

Reply:
Its intentional. I changed the common method 'getAddressFromPostcode' to accept acspData model from sdk-node. 
If I change datatype in line 32, its causing errors at other places. Once we do save and continue for unincorporated journey, this will be fixed